### ISSUE TITLE
[Reports] Enterprise fee summary report doesn't keep the date range selection

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/openfoodnetwork#10392]
## Description

When running a report and selecting a date range, this date range is usually kept after creating the report. This is convenient, if you want to make small changes to your settings and run the report again. However the enterprise fee summary report doesn't keep the date range. Instead the start and end date are empty after running the report.

## Expected Behavior

The date range should be kept after running the report.

## Actual Behaviour

The date range is empty after running the report and needs to be entered again.

## Steps to Reproduce

1. Open the enterprise fee summary report.
2. Select a date range.
3. Run the report.
4. Observe the date range.

## Animated Gif/Screenshot

![Peek 2023-02-07 19-24](https://user-images.githubusercontent.com/1790176/217376324-01411781-c62a-45d8-a818-1965dff5a347.gif)

## Workaround

Enter the date again.

## Severity
bug-s4: it's annoying, but you can use it
bug-s5: we can live with it, only a few users impacted

## Your Environment

* Version used:
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by RachL** [comment]
@drummer83 I'm removing the good first issue label here while we figure out what we want here. We want to keep the report filters after report generation, but not if we visit another page and come back (at least this is the behaviour we have on other reports) - is this correct?

Because if the date selection or another field is kept longer, we could end up with users not noticing they have selected it and end up downloading the wrong data. That's why rendering options are saved but not report filters.

**Comment by drummer83** [comment]
Yes, that's correct. Basically we want the same behavior as for all other reports.
I thought it would be easy for new devs to look at the other reports and implement the same thing here, sorry.

**Comment by RachL** [comment]
@drummer83 I'm not saying it can't be a good-first-issue, just that we need to write down the solution properly before a newby picks it up. Here "look at other reports" could lead at reproducing the behavior we have on rendering and not on filtering, so we need to align on what we want as a result and detail what "after running the report means". If we look at other report, it would be something like: 

1. Select a date range
2. Run the report
3. See the date range is still shown in the date fields
4. Refresh the page or move to another page and come back: the date field is empty again

Is this correct?

**Comment by drummer83** [comment]
Yes, that's it. Thanks!
